generate.go

```diff
@@ -52,35 +52,51 @@ type DependencyManager interface {
 
 func Generate(dependencyManager DependencyManager, logger scribe.Emitter, duringBuildPermissions DuringBuildPermissions) packit.GenerateFunc {
 	return func(context packit.GenerateContext) (packit.GenerateResult, error) {
+
 		logger.Title("%s %s", context.Info.Name, context.Info.Version)
 		logger.Process("Resolving Node Engine version")
 
+		//find the version with the highest priority
```

Comment:
There should be a space after the // in comments, ame goes for other new comments as well

Reply:
I force pushed with the changes to keep the same history